### PR TITLE
Added recyclebin cleanup command

### PR DESCRIPTION
**Pull request by fashxp**
similar to email log cleanup command. 
to automatically cleanup recyclebin entries older than x days

### CODE REVIEW

File: bundles/CoreBundle/Command/RecyclebinCleanupCommand.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    protected function configure()
+    {
+        $this
+            ->setName('pimcore:recyclebin:cleanup')
+            ->setDescription('Cleanup recyclebin entries')
+            ->addOption(
+                'older-than-days',
+                'days',
+                InputOption::VALUE_REQUIRED,
+                'Older than X Days to delete email logs'
```
Comment (ramundomario) [inline bundles/CoreBundle/Command/RecyclebinCleanupCommand.php]:
```suggestion
                'Older than X Days to delete recyclebin entries'
```

**Review by mcop1 — APPROVED** [review]

File: bundles/CoreBundle/Command/RecyclebinCleanupCommand.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+class RecyclebinCleanupCommand extends AbstractCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('pimcore:recyclebin:cleanup')
+            ->setDescription('Cleanup recyclebin entries')
+            ->addOption(
+                'older-than-days',
+                'days',
```
Comment (jdreesen) [inline bundles/CoreBundle/Command/RecyclebinCleanupCommand.php:37]: I think short options should only be a single letter.

**Comment by mcop1** [comment]
Thanks @jdreesen , follow up https://github.com/pimcore/pimcore/pull/11131

**Comment by kaurov** [comment]
Command usage sample:
./bin/console  pimcore:recyclebin:cleanup --older-than-days=60

**Pull request merged**